librz/debug/p/native/windows/windows_debug.c

```diff
@@ -988,13 +979,12 @@ int w32_dbg_wait(RzDebug *dbg, int pid) {
 			break;
 		case EXIT_PROCESS_DEBUG_EVENT:
 		case EXIT_THREAD_DEBUG_EVENT: {
-			PTHREAD_ITEM th = __find_thread(dbg, tid);
+			PTHREAD_ITEM th = find_thread(dbg, tid);
 			if (th) {
 				th->bFinished = TRUE;
-				th->hThread = INVALID_HANDLE_VALUE;
 				th->dwExitCode = de.u.ExitThread.dwExitCode;
 			} else {
-				__r_debug_thread_add(dbg, pid, tid, INVALID_HANDLE_VALUE, de.u.CreateThread.lpThreadLocalBase, de.u.CreateThread.lpStartAddress, TRUE);
```

Comment:
Is this change intended? Why?

Reply:
Yes, there should never be a situation that a thread exited without rizin having kept track of its creation. Before it happened to the initial thread, this pr fixes that